app/models/paypal_payment_id.py

```diff
@@ -0,0 +1,20 @@
+from app.models import db
+from app.models.base import SoftDeletionModel
+
+
+class PaypalPaymentId(SoftDeletionModel):
+    __tablename__ = "paypal_payment_id"
+    id = db.Column(db.String, primary_key=True)
```

Comment:
Just to obtain the paypal id, you wouldn't want to create a whole new table. You can find the paypal payment ID in the order model.

Reply:
ok thanks

Reply:
Is this  `paypal_token = db.Column(db.String)` what I'm looking for cause I don't find paypal payment ID in the order model

Reply:
If so then please review